tests/integration/genutil/gentx_test.go

```diff
@@ -222,93 +224,95 @@ func (suite *GenTxTestSuite) TestValidateAccountInGenesis() {
 			)
 
 			if tc.expPass {
-				suite.Require().NoError(err)
+				assert.NilError(t, err)
 			} else {
-				suite.Require().Error(err)
+				assert.ErrorContains(t, err, tc.expErrMsg)
 			}
 		})
 	}
 }
 
-func (suite *GenTxTestSuite) TestDeliverGenTxs() {
+func TestDeliverGenTxs(t *testing.T) {
+	t.Parallel()
+	f := initFixture(t)
+
 	var (
 		genTxs    []json.RawMessage
-		txBuilder = suite.encodingConfig.TxConfig.NewTxBuilder()
+		txBuilder = f.encodingConfig.TxConfig.NewTxBuilder()
 	)
 
 	testCases := []struct {
-		msg      string
-		malleate func()
-		expPass  bool
+		msg       string
+		malleate  func()
+		expPass   bool
+		expErrMsg string
 	}{
 		{
 			"no signature supplied",
 			func() {
-				err := txBuilder.SetMsgs(suite.msg1)
-				suite.Require().NoError(err)
+				err := txBuilder.SetMsgs(f.msg1)
+				assert.NilError(t, err)
 
 				genTxs = make([]json.RawMessage, 1)
-				tx, err := suite.encodingConfig.TxConfig.TxJSONEncoder()(txBuilder.GetTx())
-				suite.Require().NoError(err)
+				tx, err := f.encodingConfig.TxConfig.TxJSONEncoder()(txBuilder.GetTx())
+				assert.NilError(t, err)
 				genTxs[0] = tx
 			},
 			false,
+			"no signatures supplied",
 		},
 		{
 			"success",
 			func() {
-				_ = suite.setAccountBalance(addr1, 50)
-				_ = suite.setAccountBalance(addr2, 1)
+				_ = setAccountBalance(t, f, addr1, 50)
+				_ = setAccountBalance(t, f, addr2, 1)
 
 				r := rand.New(rand.NewSource(time.Now().UnixNano()))
 				msg := banktypes.NewMsgSend(addr1, addr2, sdk.Coins{sdk.NewInt64Coin(sdk.DefaultBondDenom, 1)})
 				tx, err := simtestutil.GenSignedMockTx(
 					r,
-					suite.encodingConfig.TxConfig,
+					f.encodingConfig.TxConfig,
 					[]sdk.Msg{msg},
 					sdk.Coins{sdk.NewInt64Coin(sdk.DefaultBondDenom, 10)},
 					simtestutil.DefaultGenTxGas,
-					suite.ctx.ChainID(),
+					f.ctx.ChainID(),
 					[]uint64{7},
 					[]uint64{0},
 					priv1,
 				)
-				suite.Require().NoError(err)
+				assert.NilError(t, err)
 
 				genTxs = make([]json.RawMessage, 1)
-				genTx, err := suite.encodingConfig.TxConfig.TxJSONEncoder()(tx)
-				suite.Require().NoError(err)
+				genTx, err := f.encodingConfig.TxConfig.TxJSONEncoder()(tx)
+				assert.NilError(t, err)
 				genTxs[0] = genTx
 			},
 			true,
+			"",
 		},
 	}
 
 	for _, tc := range testCases {
-		suite.Run(fmt.Sprintf("Case %s", tc.msg), func() {
-			suite.SetupTest()
+		t.Run(fmt.Sprintf("Case %s", tc.msg), func(t *testing.T) {
+			initFixture(t)
 
 			tc.malleate()
 
 			if tc.expPass {
-				suite.Require().NotPanics(func() {
+				require.NotPanics(t, func() {
```

Comment:
can this be changed to `assert.NotPanics` ?

Reply:
nope, there is no such util in assert